ozaria/site/components/teacher-dashboard/modals/ModalEditClass.vue

```diff
@@ -223,6 +223,11 @@ export default Vue.extend({
   },
 
   watch: {
+    isNewClassroom (newVal) {
```

Comment:
there is classroomInstance.isNew that can be used instead.

Reply:
Done!